docs/instrumentation.md

```diff
@@ -292,4 +292,19 @@ tpm_tis_spi.ko: tpm_tis_spi_probe()
 tpm_tis_spi.ko: tpm_tis_spi_driver_probe()
 tpm.ko: tpm_dev_release()
 tpm.ko: tpm_dev_release()
+
+```
+
+### Whole logs from the test procedure
```

Comment:
maybe replace with something like that 

### Whole logs from the test procedure

Below we have collected call stacks for Raspberry Pi TPM driver startup.

[Log file with TPM module plugged in](./logs_tpm_module/log_with_tpm_get_call_stack.txt)</br>
[Log file without TPM](./logs_tpm_module/log_without_tpm_get_call_stack.txt)</br>

Reply:
Changes applied in commit `0acd479aaffb4ea534d45381fa4415344b7cced6`